code/modules/effects/map_effects/door_helper.dm

```diff
@@ -59,3 +59,14 @@
 	"red" = list(19,38),
 	"delta" = list(19,38)
 )
+
+/obj/effect/map_effect/door_helper/lock
+//Locks/bolts any (lockable) door/airlock this marker is placed on.
```

Comment:
Properly DMDoc the marker

Also, this is called door_helper, but it seems to only work for airlocks, which are a subtype of doors, and doesn't seem to validate anywhere that it's placed over a supported door, this would likely need a maplint or unit test or a different path/name that prevents the error of it being placed on an unsupported door type

Reply:
it's the same for every door_helper in the file and i feel like implementing an entire unit test for something like this would be vastly out of scope